src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -121,15 +121,15 @@ internal static class LoggerExtension
                 new EventId(325, nameof(ScriptStartUpLoadedExtension)),
                 "Loaded extension '{startupExtensionName}' ({startupExtensionVersion})");
 
-        private static readonly Action<ILogger, Exception> _functionMetadataProviderReadingMetadata =
-            LoggerMessage.Define(LogLevel.Information,
+        private static readonly Action<ILogger, string, Exception> _functionMetadataProviderReadingMetadata =
+            LoggerMessage.Define<string>(LogLevel.Information,
                 new EventId(326, nameof(FunctionMetadataManagerLoadingFunctionsMetadata)),
-                "Reading functions metadata");
+                "Reading functions metadata from {source}");
 
-        private static readonly Action<ILogger, int, Exception> _functionMetadataProviderFunctionsFound =
-            LoggerMessage.Define<int>(LogLevel.Information,
+        private static readonly Action<ILogger, int, string, Exception> _functionMetadataProviderFunctionsFound =
+            LoggerMessage.Define<int, string>(LogLevel.Information,
                 new EventId(327, nameof(FunctionMetadataManagerFunctionsLoaded)),
```

Comment:
In reviewing this code, I noticed what appears to be a copy paste error here (before your changes). As you can see `nameof(FunctionMetadataManagerFunctionsLoaded)` is used incorrectly here. It should be `nameof(FunctionMetadataProviderFunctionFound)`, then pattern here being the name of the event maps to the log method.

You might check the others for errors as well.

Reply:
Changed!